README.md

```diff
@@ -22,9 +21,71 @@ Or install it yourself as:
 
     $ gem install microform
 
+## Generating Forms
+
+Generate a form for your model, with accompanying tests:
+
+```
+  rails g microform:form model_name
+```
+
+This will create two files: `app/forms/model_name_form.rb` and `test/forms/model_name_test.rb`.
+
 ## Usage
 
-TODO: Write usage instructions here
+ The `Microform::Submission` module provides a `submit` method that will help to better isolate your form tests. Include it in the relevant controller for your form(s):
```

Comment:
Might need to clarify that is a "controller mixin", or "provides a `submit` controller method.